test.sh

```diff
@@ -191,6 +191,7 @@ trap "print_outcome" EXIT
 settings="$(cat -- <<-EOM
     RUN:                ${RUN[@]}
     BOULDER_CONFIG_DIR: $BOULDER_CONFIG_DIR
+    GOCACHE:            $(go env | awk -F'=' '/^GOCACHE/ {print $2}' | sed "s/'//g")
```

Comment:
The following appears to work fine as well. I'm curious what this one-liner is solving.

```suggestion
    GOCACHE:            $(go env GOCACHE)
```

Reply:
I didn't know you could do that.